### DISCUSSION TITLE
Feedback from Blackhole while streaming

### DISCUSSION THREAD

**Discussion by kosz** [ExistentialAudio/BlackHole#776] [Q&A]
Hi all, 

I've recently updated my blackhole install to the latest version, as the old version was stopping my sound every 10 minutes or so while streaming. 
Unfortunately the new version, instead of stopping the sound, now will play a VERY LOUD feedback noise, that could probably physically hurt the ears of someone joining my stream. This happens intermittently, just like the silence before the update. It would work just fine, then ppl on stream would scream that there's feedback. I am considering stopping from streaming if i can't fix blackhole or find an alternative. 
Does anyone know how to fix this? 

Macbook Pro M1
16gb ram
Studio One DAW
Device Block Size 512 samples 
Traktor Kontrol S4 as sound card 
Installed using downloaded installer

**Comment by devinroth** [comment]
The issue with the old version was probably because you didn’t enable drift correction.

And with the new version you must be doing something to feedback the audio.

BlackHole is quite simple. Whatever audio you send to the output goes to the input. Nothing more or less.

I suggest you figure out your signal flow and you’ll find your feedback.

Devin

**Comment by kosz** [comment]
thanks. for the feedback I've disabled the loopback through my system sound card, and only left the loopback on my external soundcard. 
I have also enabled drift correction and that allowed me to have stable routing while streaming.

thanks for the guidance, all is well
